[17.0][MIG] sale_order_line_cancel: Migration to 17.0

Paulius11 opened the pull request.

Paulius11 committed:
Add sale_order_line_cancel from v16

Paulius11 committed:
Migrate to from v16->v17

Paulius11 committed:
Fix tests

Paulius11 committed:
Fix tests

Paulius11 committed:
[ADD] pyproject.toml

Paulius11 committed:
Update __manifest__.py

Paulius11 committed:
[FIX] Update manifest

adasatorres commented:
Hello @Paulius11, thank you for your contribution. I'm sharing a guide that I think will be useful for you during the migration: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0.
The guide explains the changes introduced in this version, provides a brief explanation on how to clone the repo and bring in the changes affecting the addon from the source version, and also describes how the commit structure should be.

rousseldenis commented:
/ocabot migration sale_order_line_cancel

rousseldenis commented:
@Paulius11 Do you plan to fix this ?

Paulius11 committed:
[MIG] sale_order_line_cancel: Migration to 17.0

- Update README.rst with 17.0 branch references
- Convert readme fragments from RST to Markdown format
- Add Paulius Gaizauskas to contributors
- Fix author field in manifest (remove GitHub handle)

Paulius11 commented:
@rousseldenis  https://github.com/OCA/sale-workflow/pull/4119
